integration-test/src/test/java/org/apache/iotdb/db/it/query/IoTDBAliasIT.java

```diff
@@ -0,0 +1,385 @@
+        "INSERT INTO root.sg2.d2(timestamp,s1,s2,s3) values(300, 45.3, 23.4, 82.0)",
+        "INSERT INTO root.sg2.d2(timestamp,s1,s2,s3) values(400, 73.4, 26.3, 83.0)"
+      };
+
+  private static final String LAST_QUERY_HEADER = "Time,timeseries,value,dataType,";
+
+  @BeforeClass
+  public static void setUp() throws Exception {
+    EnvFactory.getEnv().initBeforeClass();
+    prepareData(SQLs);
+  }
+
+  @AfterClass
+  public static void tearDown() throws Exception {
+    EnvFactory.getEnv().cleanAfterClass();
+  }
+
+  // ---------------------------------- Use timeseries alias ---------------------------------
+
+  @Test
+  public void rawDataQueryAliasTest() {
+    String expectedHeader = "Time,root.sg.d1.speed,root.sg.d1.temperature,";
+    String[] retArray =
+        new String[] {"100,10.1,20.7,", "200,15.2,22.9,", "300,30.3,25.1,", "400,50.4,28.3,"};
+
+    resultSetEqualTest("select speed, temperature from root.sg.d1", expectedHeader, retArray);
+  }
+
+  @Test
+  public void rawDataQueryWithDuplicatedColumnsAliasTest() {
+    String expectedHeader = "Time,root.sg.d1.speed,root.sg.d1.speed,root.sg.d1.s2,";
+    String[] retArray =
+        new String[] {
+          "100,10.1,10.1,20.7,", "200,15.2,15.2,22.9,", "300,30.3,30.3,25.1,", "400,50.4,50.4,28.3,"
+        };
+
+    resultSetEqualTest("select speed, speed, s2 from root.sg.d1", expectedHeader, retArray);
+  }
+
+  @Test
+  @Ignore // TODO: remove @Ignore after support alias in last query
+  public void lastQueryAliasTest() {
+    String[] retArray =
+        new String[] {"400,root.sg.d1.speed,50.4", "400,root.sg.d1.temperature,28.3"};
+
+    resultSetEqualTest(
+        "select last speed, temperature from root.sg.d1", LAST_QUERY_HEADER, retArray);
+  }
+
+  @Test
+  @Ignore // TODO: remove @Ignore after support alias in last query
+  public void lastQueryWithDuplicatedColumnsAliasTest() {
+    String[] retArray =
+        new String[] {
+          "400,root.sg.d1.speed,50.4", "400,root.sg.d1.s1,50.4", "400,root.sg.d1.s2,28.3"
+        };
+
+    resultSetEqualTest(
+        "select last speed, s1, speed, s2 from root.sg.d1", LAST_QUERY_HEADER, retArray);
+  }
+
+  @Test
+  public void aggregationQueryAliasTest() {
+    String expectedHeader =
+        "count(root.sg.d1.speed),count(root.sg.d2.speed),max_value(root.sg.d1.temperature),"
+            + "max_value(root.sg.d2.temperature),";
+    String[] retArray = new String[] {"4,4,28.3,26.3,"};
+
+    resultSetEqualTest(
+        "select count(speed), max_value(temperature) from root.sg.*", expectedHeader, retArray);
+  }
+
+  @Test
+  public void alterAliasTest() {
+    String[] retArray = {"100,80.0,", "200,81.0,", "300,82.0,", "400,83.0,"};
+
+    try (Connection connection = EnvFactory.getEnv().getConnection();
+        Statement statement = connection.createStatement()) {
+      statement.execute("ALTER timeseries root.sg.d2.s3 UPSERT ALIAS='powerNew'");
```

Comment:
This is a write request, please make sure whether it will influence other test cases, as the environment setup uses @BeforeClass .